docs/source/howto/codes.rst

```diff
@@ -162,6 +386,19 @@ Adding support for a custom transport
 `#3990`_
 
 
+.. |Int| replace:: :py:class:`~aiida.orm.nodes.data.int.Int`
+.. |Code| replace:: :py:class:`~aiida.orm.nodes.data.Code`
+.. |Parser| replace:: :py:class:`~aiida.parsers.parser.Parser`
+.. |parse| replace:: :py:class:`~aiida.parsers.parser.Parser.parse`
+.. |folder| replace:: :py:class:`~aiida.common.folders.Folder`
+.. |folder.open| replace:: :py:class:`~aiida.common.folders.Folder.open`
+.. |CalcJob| replace:: :py:class:`~aiida.engine.processes.calcjobs.calcjob.CalcJob`
+.. |CalcInfo| replace:: :py:class:`~aiida.common.CalcInfo`
+.. |CodeInfo| replace:: :py:class:`~aiida.common.CodeInfo`
+.. |spec| replace:: ``spec``
+.. |define| replace:: :py:class:`~aiida.engine.processes.calcjobs.CalcJob.define`
+.. |prepare_for_submission| :py:class:`~aiida.engine.processes.calcjobs.CalcJob.prepare_for_submission`
```

Comment:
I think sphinx might not like the underscore, but not sure.

Reply:
Nah, it was the fact that I was not telling it to actually `replace::` the thing =P.